src/components/CardDeck/styles.module.scss

```
@@ -37,6 +37,9 @@
 
 .item {
     display: flex;
+    // A deck item is a single vertical page: stack its cards, never let two
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)